src/internal/components/token-list/index.tsx

```diff
@@ -0,0 +1,56 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+import React, { useState } from 'react';
+import clsx from 'clsx';
+
+import { SelectToggle } from './select-toggle';
+import styles from './styles.css.js';
+import { I18nStrings } from './interfaces';
+import InternalSpaceBetween from '../../../space-between/internal';
+import { useUniqueId } from '../../hooks/use-unique-id';
+
+interface TokenListProps<Item> {
```

Comment:
minor: If you already have an `interfaces.ts`, you might as well put the Props interface in there